BukkyOmo/Questioner
# #163740821 Implement  fetch API for signin page

#### What does this PR do?
This PR:
* Implement Fetching API for signin endpoint.

#### Description of Task to be completed
* Create a js file for signup.
* Include the signup page heroku page url.
* Create variables for each field that is required to be entered by the user and get them by their id.
* Create an event listener for the button to be clicked. 
* Create your fetch which should include method of http request, body and the headers.
* Write the fetch logic to save the token gotten from user in local storage.

#### How should this be manually tested?
* Clone this repository
* Check out to <Implement  fetch API for signin page-#163740821> branch
* Check signin.html page via browser.

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories? (if applicable)
#163740821
